.github/workflows/sim-test.yaml

```
@@ -0,0 +1,103 @@
+name: Simulator Test
+
+on: [pull_request]
+
+jobs:
+  build-check:
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE: github-actions-etrobocon2021
+      APP_NAME: etrobocon2021
+
+    steps:
+      # ${{ github.workspace }} を使用するために必要
+      - name: Checkout
+        uses: actions/checkout@v1
```

```suggestion
        uses: actions/checkout@v2
```
ここは特に理由がなければv2がよさそうです．最新が最高なので！

ref. https://github.com/marketplace/actions/checkout